server/pgClient.js

```diff
@@ -0,0 +1,77 @@
+const client = require('../database/pgConnect.js');
+
+exports.getAll = (req, res) => {
+  let query = `SELECT * FROM records WHERE id=${req.params.id}`;
+  client.query(query, (error, results) => {
+    if (error) {
+      return console.log('error happened', error);
+    }
+    var result = [{
+      packaging: {
+        measurments: {
+          width: results.rows[0].width,
+          height: results.rows[0].height,
+          length: results.rows[0].length,
+          packages: results.rows[0].packages
+        }
+      },
+      sizes: {
+        attributes: {
+          "thread-count": results.rows[0].threadcount,
+          "Pillowcasse quantity": results.rows[0].pillowcasequantity,
+          "Duvet cover length": results.rows[0].duvetcoverlength,
+          "Duvet cover width": results.rows[0].duvetcoverwidth,
+          "Pillowcase length": results.rows[0].pillowcaselength,
+          "Pillowcase width": results.rows[0].pillowcasewidth
+        },
+        fitting: results.rows[0].fitting
+      },
+      imageUrls: results.rows[0].imagesurls,
+      name: results.rows[0].name,
+      id: results.rows[0].id,
+      description: results.rows[0].description,
+      articleNumber: results.rows[0].articlenumber,
+      details: results.rows[0].details,
+      materials: results.rows[0].materials,
+      sustainibility: results.rows[0].sustainability
+    }]
+    return res.send(result);
+  })
+};
+
+exports.add = (req, res) => {
+  const query = `INSERT INTO records (id,name,description,articleNumber,details,materials,sustainability,width,height,length,weight,packages,shortDesc,threadCount,pillowcaseQuantity,duvetCoverLength,duvetCoverWidth,pillowcaseLength,pillowcaseWidth,fitting,imagesUrls) VALUES (${req.body.id},'${req.body.name}','${req.body.description}','${req.body.articleNumber}','${req.body.details}','${req.body.materials}','${req.body.sustainability}',${req.body.width},${req.body.height},${req.body.length},${req.body.weight},${req.body.packages},'${req.body.shortDesc}',${req.body.threadCount},${req.body.pillowcaseQuantity},${req.body.duvetCoverLength},${req.body.duvetCoverWidth},${req.body.pillowcaseLength},${req.body.pillowcaseWidth},'${req.body.fitting}', ARRAY${req.body.imagesurls});`
+
+  client.query(query, (error, results) => {
+    if (error) {
+      console.log('could not post', error)
+      return res.status(404).send();
+    }
+    console.log('posted')
+    return res.status(200).send();
+  })
+}
+
+// {
+//   "id": 19,
```

Comment:
probably delete this unless you need it for debugging (add a note that its for debugging?)